cpp/include/Ice/LocalException.h

```diff
@@ -16,7 +16,13 @@ namespace Ice
     class ICE_API LocalException : public Exception
     {
     public:
-        using Exception::Exception;
+        /**
```

Comment:
All derived local exceptions must provide a message, meaning that for local exceptions, ice_id() is never used as the fallback message.